SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.cpp

```diff
@@ -42,52 +42,48 @@ void ARegisterAutoDestroyActorsTestPart1::PrepareTest()
 		AddStep(
 			TEXT("CLIENT_ALL_CheckActorsSpawned"), FWorkerDefinition::AllClients, nullptr, nullptr,
 			[this](float DeltaTime) {
-				int NumCharactersFound = 0;
-				int NumCharactersExpected = GetNumberOfServerWorkers();
+				int32 NumCharactersFound = 0;
+				int32 NumCharactersExpected = GetNumberOfServerWorkers();
 				UWorld* World = GetWorld();
-				for (TActorIterator<ACharacter> It(World); It; ++It)
+				for (ACharacter* Character : TActorRange<ACharacter>(World))
```

Comment:
Very very minor: but unreal has some stl like algo's, in this case the mildly awkward CountIf will work. (No unconditional count is available sadly).
```
const int32 NumCharactersFound = Algo::CountIf(TActorRange<ACharacter>(World), [](){ return true;}); 
```

Reply:
Tried it, but unfortunately you also have to take the first argument in the predicate (even if you don't use it), and our auto-formatter spreads it across two lines, so I think the reading complexity is the same as the for-each case

Reply:
This gives more const and is a bit more explicit in its intent (it uses a function called Count) but very minor so fine as is.